src/components/organisms/Tooltip/Tooltip.tsx

```diff
@@ -1,20 +1,29 @@
-import { FC, ReactElement, SVGProps } from 'react'
+import { FC, SVGProps } from 'react'
 import BaseButton from 'components/atoms/BaseButton/BaseButton'
 import { showModal, ModalTypes } from 'components/organisms/modals/ModalRoot'
 import { IconProps } from 'components/molecules/Button/Button'
 import { ReactComponent as QuestionMark } from 'assets/icons/question_mark.svg'
 import classNames from 'classnames'
+import { useTranslation } from 'react-i18next'
+import ReactHtmlParser from 'html-react-parser'
+import helpSections from 'pages/Manual/helpSections.json'
 
 import classes from './classes.module.scss'
-import { useTranslation } from 'react-i18next'
+
+export interface HelpSections {
+  [key: string]: {
+    title: string
+    content: string
+    tooltipContent: string
```

Comment:
**[non-blocker]**

Let's have "tooltipContent" as an optional field. We might have sections in the manual that won't have a matching tooltip, in which case they also might not have "tooltipContent"